src/theme/helpers/getStylesByBrand.ts

```diff
@@ -4,7 +4,12 @@ export const recursiveSmooshTwoBrands = (obj1: StyleDefinition, obj2: StyleDefin
   // recursively map so that overrides smooshes into baseStyles
   const simpleValueTypes = ['string', 'number', 'boolean'];
   const end: StyleDefinition = { ...obj1 };
-  Object.entries(obj2).forEach(([key, value]) => {
+  const obj2Entries = Object.entries(obj2)
+
+  let i = 0;
+  obj2Entries.forEach(([key, value]) => {
```

Comment:
`forEach` has a second parameter, and that's the index. So you don't need to make a new `i` variable to track if for you -- you can use the second parameter.

Reply:
lol i had a feeling there was a better way. of course